dev-games/libsmacker/libsmacker-1.1.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
```

Comment:
Where are you using eutils?

Reply:
Because I need to invoke autoreconf --install

Reply:
`eautoconf` is provided by autotools.

Reply:
it did not work without...

Reply:
but that was becasue I had eutils first.. removed it.